deploy/examples/create-external-cluster-resources.py

```diff
@@ -590,9 +589,12 @@ def _invalid_endpoint(self, endpoint_str):
         try:
             if endpoint_str_ip[0] == "[":
                 endpoint_str_ip = endpoint_str_ip[1 : len(endpoint_str_ip) - 1]
-            ip_type = (
-                "IPv4" if type(ip_address(endpoint_str_ip)) is IPv4Address else "IPv6"
-            )
+            ip_type = type(ip_address(endpoint_str_ip))
+            if ip_type == IPv4Address:
+                ip_type = "IPv4"
+            else:
+                ip_type = "IPv6"
+
```

Comment:
old code version is more pythonic! not sure why are we changing this. Is this causing any warning/error?